src/test/unit_tests/jest_tests/test_nc_upgrade_manager.test.js

```diff
@@ -673,6 +726,101 @@ describe('nc upgrade manager - upgrade config directory', () => {
             assert_upgrade_finish_or_fail_data(system_json_data, expected_data);
         });
     });
+
+    describe('nc upgrade manager - upgrade_config_dir', () => {
+        beforeEach(async () => {
+            await create_config_dir(config.NSFS_NC_DEFAULT_CONF_DIR);
+            await create_config_dir(config_root);
+            await create_redirect_file(config_fs, config_root);
+        });
+
+        afterEach(async () => {
+            await clean_config_dir(config_fs, config_root);
+        }, TEST_TIMEOUT);
+
+        it('upgrade_config_dir - empty system_data', async () => {
+            const expected_version = pkg.version;
+            const hosts_list = [hostname];
+            const expected_err_msg = 'system does not exist';
+            await expect(nc_upgrade_manager.upgrade_config_dir(expected_version, hosts_list))
+                .rejects.toThrow(expected_err_msg);
+        });
+
+        it('upgrade_config_dir - config_directory property is missing in system.json', async () => {
```

Comment:
Where do we see that it is missing? could you add a comment where we omitted it?
(I know in the [docs](https://github.com/noobaa/noobaa-core/blob/master/docs/NooBaaNonContainerized/Upgrade.md) we have the "Additional Upgrade Properties of system.json" and we expect "New config directory information".
Maybe we can accurate the name of "current_expected_system_json" because it is confusing (the current is version 5.17.0, but when I read it in 5.18.0 I think it is the current...).

Reply:
config_directory property is missing in current_expected_system_json, you only have RPM information, nothing about config_directory

Reply:
Could you add a comment about this?